ruby_programming/git/lesson_a_deeper_look_at_git.md

```diff
@@ -30,13 +30,23 @@ Let's look at some ways we can change recent and distant history to fit our need
 - Squash commits together
 - Split up commits 
 
+#### Getting Set Up
+
+Before we get started with the lesson, let's create a git playground in which we can safely follow along with the code and perform history changing operations. Go to GitHub, and as you have in the past create a new repository. Call it whatever you'd like, and clone this repository on your local system. Now, let's `cd` into the repository we just cloned, and create some new files! Once you're in the repository follow along with the following commands. Look them up if you're confused about anything that's happening - the more practice we get using our Linux Terminal the better. 
+
+~~~bash
+$ touch test{1..4}.md
+$ git add test1.md && git commit -m 'Create first file'
+$ git add test2.md && git commit -m 'Create second file'
+$ git add test3.md && git commit -m 'Create third file and create fourth file'
```

Comment:
What would you think about using `git commit -am` here rather than doing conjoined (but separate) stages and commits?

Reply:
So my thinking was that the separate stages/commits are necessary to give learners three separate commits in their git history to play around with.

I could be misunderstanding, but wouldn't `git commit -am` add all of the files at once, which in turn would mean we would only have 1 commit to work with when we get to the **Changing Multiple Commits** section?